src/main/java/io/jenkins/plugins/jfrog/callables/JFrogCliDownloader.java

```diff
@@ -48,6 +48,8 @@ public class JFrogCliDownloader extends MasterToSlaveFileCallable<Void> {
 
     @Override
     public Void invoke(File toolLocation, VirtualChannel channel) throws IOException, InterruptedException {
+        log.getLogger().println("[JFrogCliDownloader] Starting CLI download");
```

Comment:
why are we downloading jfrog-cli on every stage?
cant this be downloaded once and used every where? 
or if we can check jf already exists and only download when it is not present that should me efficient right instead of multuple jfrog-cli downloads?